.github/workflows/jira_issue.yml

```diff
@@ -11,20 +11,25 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: create
-        id: create
-        uses: ./.github/actions/gajira-create
+
+      - name: Login
+        uses: atlassian/gajira-login@master
         env:
-          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
           JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
           JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Create
+        id: create
+        uses: atlassian/gajira-create@master
         with:
-          PROJECT: SNOW
-          TYPE: Bug
-          AREA: "Data Platform: Ecosystem"
-          ASSIGNEE: "5d56d115ba0a060da220202b"
-          SUMMARY: "{{ event.issue.title }}"
-          DESCRIPTION: "{{ event.issue.body }}\n\n_Created from GitHub Action_ for {{ event.issue.html_url }}"
+          project: SNOW
+          issuetype: Bug
+          summary: '${{ github.event.issue.title }}'
+          description: |
+            '${{ github.event.issue.body }} \\ \\ _Created from GitHub Action_ for ${{ github.event.issue.html_url }}'
+          fields: '{ "customfield_11401": {"id": "12360"}, "assignee": "5d56d115ba0a060da220202b" }'
```

Comment:
I ended up having to use this ID, using the name wasn't working:
Here is a list of the Data Platform related Areas from JIRA:
```
12359[Data Platform: Data Lake],
12358[Data Platform: Data Pipelines], 
12360[Data Platform: Ecosystem], 
12327[Data Platform: Extensibility], 
12365[Data Platform: ML Engineering]
```